k8s/conf/vppagent-icmp-responder-nse.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+spec:
+  template:
+    metadata:
+      labels:
+        networkservicemesh.io: "true"
+        networkservicemesh.io/app: "vppagent-icmp-responder-nse"
+    spec:
+      hostPID: true
+      containers:
+        - name: icmp-responder-nse
+          securityContext:
+            privileged: true
+          image: networkservicemesh/vppagent-icmp-responder-nse:latest
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              nsm.ligato.io/socket: 1
+        - name: vppagent-nse
+          securityContext:
+            privileged: true
+          image: networkservicemesh/vppagent-nse:latest
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: INITIAL_LOGLVL
+              value: debug
+          volumeMounts:
+            - name: workspace
+              mountPath: /var/lib/networkservicemesh
```

Comment:
NSE's and NSC's can't mount /var/lib/networkservicemesh in their Pod specs.  Their workspaces get mounted there by the DevicePlugin API mechanism.  See https://github.com/ligato/networkservicemesh/blob/master/k8s/conf/icmp-responder-nse.yaml#L17

Reply:
This mount is created only for vppagent container. Is it ok?